packages/react/src/components/form/Checkbox/useCheckboxGroup.ts

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useId, useRef, useState } from 'react';
+import type { ChangeEvent, ReactNode } from 'react';
+
+export type UseCheckboxGroupProps = {
+  disabled?: boolean;
+  error?: ReactNode;
+  name?: string;
+  readOnly?: boolean;
+  value?: string[];
+  onChange?: (
```

Comment:
Suggest updating these with jsdoc comments on usage and `@default` value? Best to document now that its fresh.